packages/icons/README.md

```diff
@@ -20,4 +20,26 @@ or
 npm install --save @lmc-eu/spirit-icons
 ```
 
+## Usage
+
+### SVG files
+
+You can use SVG files directly from `@lmc-eu/spirit-icons/svg` directory by importing them or copying them to your app structure.
+
+### React
```

Comment:
Maybe we should mention we use the pascal case for React?

Reply:
This is fixed by pascalcasing name prop.